posts/2022-04-15-libertydevops-gitjenkinshelmiks.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/shamjithantholi
+author_github: https://github.com/shamjithantholi
+seo-title: TITLE - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: "A high level explanation about how to build a devops pipeline with Jenkins as the core controller component "
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins
```

Comment:
The title needs to be shorter but still search engine friendly. Based on YK's comments, how about:
"Continuous integration and delivery (CI/CD) of cloud-native Java applications using Jenkins"
or
"Using Jenkins for continuous integration and delivery (CI/CD) of cloud-native Java applications"

@yeekangc ?

Reply:
modified title to "Continuous integration and delivery (CI/CD) of cloud-native Java applications using Jenkins"